### PR TITLE
Update Kubernetes version requirement and advice user to use latest KinD version

### PR DESCRIPTION
**Pull request by mikeshng**
Signed-off-by: Mike Ng <ming@redhat.com>

for https://github.com/open-cluster-management-io/clusteradm/issues/251

### CODE REVIEW

File: content/en/getting-started/quick-start/_index.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ To try out the cluster registration, let's walk through the following steps:
    ```shell
    # By default, it installs the latest release of the OCM components.
    # Use e.g. "--bundle-version=latest" to install latest development builds.
+   # NOTE: For hub cluster version between v1.16 to v1.18 use the parameter: --use-bootstrap-token
```
Comment (qiujian16) [inline content/en/getting-started/quick-start/_index.md:104]: v1.19 based on my testing

Comment (mikeshng) [reply content/en/getting-started/quick-start/_index.md]: Done as suggested.

**Pull request merged**